README.md

```diff
@@ -0,0 +1,142 @@
+# Processing Chain for COSMO and ICON Simulations
+
+[![Build Status](https://jenkins-mch.cscs.ch/buildStatus/icon?job=processing-chain-weekly)](https://jenkins-mch.cscs.ch/view/C2SM/job/processing-chain-weekly/)
+[![Test Status](https://jenkins-mch.cscs.ch/buildStatus/icon?job=processing-chain-weekly&config=test)](https://jenkins-mch.cscs.ch/view/C2SM/job/processing-chain-weekly/)
+
+The Processing Chain is a python script that prepares necessary input
+data, submits compute-jobs to the queue on Piz Daint and does
+post-processing steps. It supports different types of simulations,
+including **COSMO**, **COSMO-GHG**, **COSMO-ART**, **ICON** and
+**ICON-ART**. The chain can flexibly be adapted according to your needs,
+e.g., by creating your own case or adding new jobs.
+
+## Environment Setup
+
+The following steps allow you to create and use your own virtual
+environment to run the Processing Chain. We recommend to use a conda
+environment for the usage of the provided scripts. Please follow the
+instruction for the installation.
+
+### 1\. Install Miniconda
+
+Install as user specific miniconda, e.g. on `/scratch` (enter `cd
+$SCRATCH` and `pwd` at the command line to get to your personal scratch
+directory on Daint). When the command prompt asks for installation
+location, provide the path to your scratch and append `/miniconda3`.
+
+> **Note**: The default location would be on your /home directory, which
+> may lead to memory issues. (the default location would be on your home
+> directory, which may lead to memory issues).
+
+To install the latest miniconda, type:
+
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    bash Miniconda3-latest-Linux-x86_64.sh
+
+Then, export `$PATH` to your conda installation:
+
+    export PATH="$SCRATCH/miniconda3/bin:$PATH"
+
+### 2\. Create the Conda Environment
+
+Create a conda environment `proc-chain` with and install requirements:
+
+    conda env create -f env/environment.yml
+
+Activate the environment (use "source activate" in case "conda activate"
+does not work):
+
+    conda activate proc-chain
+
+If you already have the environment but want to update it:
+
+    conda env update --file env/environment.yml --prune
+
+### 3\. Store user-specific data
+
+To register your email address and standard account, store them into
+these files within your home directory:
+
+    echo <your_account_id> > ~/.acct
+    echo <your_email_address> > ~/.forward
+
+## Run the Chain
+
+Once everything has been set up correctly according to the above steps,
+you just need to execute the following command to activate your
+environment:
+
+    conda activate proc-chain
+
+To test if your environment has been successfully set, use the command
+line help to see the available arguments for the main script:
+
+    python run_chain.py -h
+
+To run the example cases with their standard jobs, you will need to
+download the necessary input data first (this may take some time):
```

Comment:
Maybe also here mention that it's better to run this on `$SCRATCH`?

Reply:
It is highly recommended now to clone the processing chain on `$SCRATCH` (as you would do for model source code). I will add this to the README. Taking track of your own changes / cases etc. should be done within your own fork.